packages/slice-machine/src/modules/environment/index.ts

```diff
@@ -8,7 +8,7 @@ import {
 } from "@models/common/Environment";
 import Warning from "@models/common/Warning";
 import { ConfigErrors } from "@models/server/ServerState";
-import { Framework } from "@models/common/Framework";
+import { Frameworks } from "@slicemachine/core/build/src/models/Framework";
```

Comment:
Import type instead of importing built file no ?

Reply:
Can't for now :x, as we use Frameworks.something, we have to target the build for now.
Used in `selectIsPreviewAvailableForFramework`